traveler-ios-ui/TravelerKitUI/ViewControllers/BookingItemSearch/BookingItemSearchSortFilterViewController.swift

```diff
@@ -0,0 +1,195 @@
+        }
+    }
+        
+    open override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.register(UINib(nibName: "BookingItemSearchSortFilterCell", bundle: Bundle(for: BookingItemSearchSortFilterViewController.self)), forCellReuseIdentifier: cellIdentifier)
+        
+        resetData()
+    }
+    
+    open override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        switch (segue.identifier, segue.destination) {
+        case ("sortSegue", let navVC as UINavigationController):
+            let vc = navVC.topViewController as? BookingItemSearchSortViewController
+            vc?.delegate = self
+            vc?.currentSelection = currentSort!
+        case ("filterPriceSegue", let navVC as UINavigationController):
+            let vc = navVC.topViewController as? BookingItemSearchFilterPriceViewController
+            vc?.delegate = self
+            vc?.currentRange = currentRange
+            vc?.fullRange = fullRange
+        default:
+            Log("Unknown segue", data: segue, level: .warning)
+            break
+        }
+    }
+    
+    @IBAction func cancel() {
+        dismiss(animated: true, completion: nil)
+    }
+    
+    @IBAction func submit() {
+        guard let currentSearchResult = currentSearchResult, searchQuery != nil else {
+            cancel()
+            return
+        }
+        
+        let sortData = currentSort!.data
+        var bookingItemSearchFilters = BookingItemSearchFilters()
+        bookingItemSearchFilters.priceRange = PriceRangeFilter(range: currentRange!, currency: currentSearchResult.facets.minPrice.baseCurrency)
+        bookingItemSearchFilters.categories = searchQuery!.categories
+        bookingItemSearchFilters.city = searchQuery!.city
+        bookingItemSearchFilters.country = searchQuery!.country
+        bookingItemSearchFilters.sortOption = sortData.sortOption
+        bookingItemSearchFilters.sortOrder = sortData.sortOrder
+        searchQuery = searchQuery!.filterSearchWith(bookingItemSearchFilters)
+        delegate?.bookingItemSearchSortFilterController(self, didSubmit: searchQuery)
+    }
+    
+    func resetData() {
+        currentSort = defaultSort
+        
+        if let minFacetsPrice = currentSearchResult?.facets.minPrice.valueInBaseCurrency, let maxFacetsPrice = currentSearchResult?.facets.maxPrice.valueInBaseCurrency, minFacetsPrice <= maxFacetsPrice {
+            currentRange = minFacetsPrice...maxFacetsPrice
+        } else {
+            currentRange = fullRange
+        }
+    }
+    
+    @IBAction func reset() {
+        resetData()
+        tableView.reloadData()
+    }
+    
+    public override func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+    
+    public override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if section == 0 {
+            return 1
+        } else {
+            return 1
+        }
```

Comment:
It always returns 1 so the if statement is not needed? Or add a TODO comment for future changes.